pkgs/development/ocaml-modules/ppx_deriving_cmdliner/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, buildDunePackage
+, fetchFromGitHub
+, alcotest
+, cmdliner
+, cppo
+, ppx_deriving
+, ppx_tools
+, ppxfind
+, ppxlib
+}:
+
+buildDunePackage rec {
+  pname = "ppx_deriving_cmdliner";
+  version = "0.5.1";
+
+  minimumOCamlVersion = "4.03";
```

Comment:
In this accurate? I can’t build it unless OCaml is at version 4.10.

Reply:
As of today, ppx_deriving_cmdliner has switched to ppxlib and thus requires at least OCaml 4.05. See release [v0.6.0](https://github.com/hammerlab/ppx_deriving_cmdliner/releases/tag/v0.6.0).

Reply:
@trepetti thanks for the new version.

Reply:
Thanks for packaging it, @romildo. I've been thinking of trying to use Nix for OCaml instead of my current method of mixing it with an OPAM flow. It's great to keep adding as many libraries as possible to both.